apps/src/sites/studio/pages/lessons/show.js

```diff
@@ -2,16 +2,39 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import getScriptData from '@cdo/apps/util/getScriptData';
 import LessonOverview from '@cdo/apps/templates/lessonOverview/LessonOverview';
-import {sampleActivities} from '../../../../../test/unit/lib/levelbuilder/lesson-editor/activitiesTestData';
 import announcementsReducer, {
   addAnnouncement
 } from '@cdo/apps/code-studio/announcementsRedux';
 import {getStore} from '@cdo/apps/code-studio/redux';
 import {registerReducers} from '@cdo/apps/redux';
 import {Provider} from 'react-redux';
+import _ from 'lodash';
 
 $(document).ready(function() {
   const lessonData = getScriptData('lesson');
+  const activities = lessonData['activities'];
+
+  // Rename any keys that are different on the backend.
+  activities.forEach(activity => {
+    activity.displayName = activity.name || '';
+    delete activity.name;
+
+    activity.duration = activity.duration || 0;
+
+    activity.activitySections.forEach(activitySection => {
+      activitySection.displayName = activitySection.name || '';
+      delete activitySection.name;
+
+      activitySection.text = activitySection.description || '';
```

Comment:
Do we want to move to using description instead of text here? (I think maybe we talked about this before but can't remember)

Reply:
yes, we want to move to using description instead of text here. added here: [PLAT-415]